packages/api/db/tables.sql

```diff
@@ -131,6 +150,22 @@ CREATE TABLE IF NOT EXISTS upload
     UNIQUE (user_id, source_cid)
 );
 
+CREATE VIEW admin_search as
+select
+  u.id as user_id,
+  u.github_id as username,
+  u.email as email,
+  ak.secret as token,
+  ak.id as token_id,
+  ak.deleted_at as deleted_at,
+  akh.inserted_at as reason_inserted_at,
+  akh.reason as reason,
+  akh.status as status
+from public.user u
+right join auth_key ak on ak.user_id = u.id
+full outer join auth_key_history akh on akh.auth_key_id = ak.id
+where akh.deleted_at is null;
```

Comment:
i couldn't find a way to `.or()` across multiple tables in a single postrest-js query, and the postrest docs recomment adding a view for more complicated filtering: https://postgrest.org/en/stable/api.html#operators